lib/beaker/subcommands.rb

```diff
@@ -0,0 +1,23 @@
+require "thor"
+require "fileutils"
+require_relative "shared/subcommands_util"
+
+module Beaker
+  class Subcommands < Thor
+    include Beaker::Shared::SubcommandsUtil
```

Comment:
i'm not too sure if i need this include and the require_relative, i was basically trying to mimic static method imports in java, so subcommands.rb only contains subcommand methods